backend/tests/integration/records/views/test_views_conferences.py

```diff
@@ -175,10 +175,13 @@ def test_conferences_sort_options(api_client, db, es, create_record):
     response_data_sort_options = response_data["sort_options"]
 
     expected_status_code = 200
-    expected_sort_options_1 = {"value": "mostrecent", "display": "Most Recent"}
+    expected_sort_options = [
+        {"display": "Date ascending", "value": "dateasc"},
+        {"display": "Date descending", "value": "datedesc"},
+    ]
 
     assert expected_status_code == response_status_code
-    assert expected_sort_options_1 in response_data_sort_options
+    assert expected_sort_options == response_data_sort_options
```

Comment:
maybe this will became a flaky test, the order in the response it's not always guaranteed